examples/default/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -12,4 +12,12 @@
             <certificates src="user" />
         </trust-anchors>
     </base-config>
+
+    <debug-overrides>
+        <trust-anchors>
+            <!-- Trust user added CAs while debuggable only -->
+            <certificates src="user" />
+            <certificates src="system" />
+        </trust-anchors>
+    </debug-overrides>
```

Comment:
I think this is doing the same thing `base-config` before it is doing, so I assume we can safely remove it. Is there anything I'm missing?

Reply:
forgot to remove it 😅✋